src/oatpp/core/data/mapping/type/Primitive.hpp

```diff
@@ -123,6 +126,11 @@ class String : public type::ObjectWrapper<std::string, __class::String> {
     return this->m_ptr.operator*();
   }
 
+  operator std::string() const
```

Comment:
Since it's an implicit conversion with implicit dereferencing
I suggest adding `throw` in case of `m_ptr == nullptr`

Reply:
What should be thrown in this case? Would `std::runtime_error` be fine?

Reply:
Yes, `std::runtime_error` is the common ground we chose. Just take a look at the other exceptions. They are most of the time

`std::runtime_error("[<full qualified classname with namespace>] Error: <message>")`

Reply:
Done.